### PR TITLE
CosmosDB Intial Integration Test

### PR DESCRIPTION
**Pull request by vuqtran88**
## Description
This PR lays ground work for testing Cosmos DB instrumentation. More specific tests will be added in later PRs.

The integration test secret need to be updated with the new CosmosDB server secret

```
      "CosmosDBTests": {
        "CustomSettings": {
          "AuthKey": "your CosmosDB secret",
          "Server": "you CosmosDB endpoint"
        }
      }      
```

### CODE REVIEW

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        protected CosmosDBTestsBase(TFixture fixture, ITestOutputHelper output)  : base(fixture)
+        {
+            _fixture = fixture;
+            _fixture.TestLogger = output;
+
+            _fixture.SetTimeout(TimeSpan.FromMinutes(2));
+
+            _fixture.AddCommand($"CosmosDBExerciser StartAgent");
+            _fixture.AddCommand($"CosmosDBExerciser CreateReadAndDeleteDatabase test_db_{Guid.NewGuid():x}");
```
Comment (JcolemanNR) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs:30]: Nice, I like the dynamic aspect of the db name!

**Review by JcolemanNR — CHANGES_REQUESTED** [review]
Please update the checked in example secretes with the default values

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs (hunk 1 above)
Comment (nr-ahemsath) [reply tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs:30]: example-secrets.json has been updated with the CosmosDB local test values

**Review by nr-ahemsath — APPROVED** [review]
Looks good to me.  I made sure the tests built and passed locally.  Approved.

**Pull request merged**